src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeaderArtwork.tsx

```diff
@@ -30,6 +30,8 @@ export const DefaultHeaderArtwork: React.FC<DefaultHeaderArtworkProps> = ({
 }) => {
   const { trackEvent } = useTracking()
 
+  if (!artwork.node.image) return null
```

Comment:
Curious what the UI treatment of this case is?

Reply:
Just skips it.

Reply:
Yeah but does it mean there's no header image at all? (I'm not super familiar with how this works-- mind attaching a screenshot?)

Reply:
Oh do you mean if the entire rail has no images? — I actually didn't test for that, will get back to this in a sec.

Reply:
Ohh I see, this is just one image of many! This change seems 👍 in any case.

Reply:
Yeah, I the naming of all these components leaves a lot to be desired.